<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ct_FormProdutos</class>
 <widget class="QFrame" name="ct_FormProdutos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LC SISTEMAS | EDITAR PRODUTO</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icones/edit.jpg</normaloff>../icones/edit.jpg</iconset>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QFrame" name="fr_FormProdutos">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>500</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFF;
border: none</string>
   </property>
   <widget class="QLabel" name="lb_FormProdutos">
    <property name="geometry">
     <rect>
      <x>419</x>
      <y>10</y>
      <width>561</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 14px;
font-family: &quot;Arial&quot;;
font-weight: bold;

border-bottom: 2px solid #A2A2A2
}</string>
    </property>
    <property name="text">
     <string>EDITAR PRODUTO</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tx_idProduto">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background: #CFCFCF;
border: 1px solid #A2A2A2;
color: #000;
font-size: 14px;
font-family: &quot;Arial&quot;;
font-weight: bold;

}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_FotoProduto">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>331</width>
      <height>391</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Insira uma imagem</string>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #A2A2A2;
background: url('Images/icon/Photo.svg') center no-repeat 
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_FormProdutos_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>60</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 12px;
font-family: &quot;Arial Unicode MS&quot;;
font-weight: bold;
color: #797979
}</string>
    </property>
    <property name="text">
     <string>DESCRIÇÃO</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tx_DescricaoProduto">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>85</y>
      <width>450</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background: #CFCFCF;
border-radius: 2px;
color: #000;
font: 13px &quot;Arial&quot; ;

}
QLineEdit:Focus {
border: 1px solid red;
}</string>
    </property>
    <property name="placeholderText">
     <string>Descrição Produto</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lb_FormProdutos_3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>120</y>
      <width>215</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 12px;
font-family: &quot;Arial Unicode MS&quot;;
font-weight: bold;
color: #797979
}</string>
    </property>
    <property name="text">
     <string>CATEGORIA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_AddCategoriaProduto">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>145</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background: #7AB32E;
color: #FFF
}
QPushButton:hover{
background-color: #40a286
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/add/Sem.png</normaloff>:/add/Sem.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="lb_FormProdutos_4">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>120</y>
      <width>215</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 12px;
font-family: &quot;Arial Unicode MS&quot;;
font-weight: bold;
color: #797979
}</string>
    </property>
    <property name="text">
     <string>MARCA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_AddMarcaProduto">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>145</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background: #7AB32E;
color: #FFF
}
QPushButton:hover{
background-color: #40a286
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/add/Sem.png</normaloff>:/add/Sem.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="lb_FormProdutos_5">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>180</y>
      <width>215</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 12px;
font-family: &quot;Arial Unicode MS&quot;;
font-weight: bold;
color: #797979
}</string>
    </property>
    <property name="text">
     <string>ESTOQUE</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_FormProdutos_6">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>180</y>
      <width>215</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 12px;
font-family: &quot;Arial Unicode MS&quot;;
font-weight: bold;
color: #797979
}</string>
    </property>
    <property name="text">
     <string>CÓDIGO</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tx_EstoqueMinimoProduto">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>205</y>
      <width>215</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background: #CFCFCF;
border-radius: 2px;
color: #000;
font: 13px &quot;Arial&quot; 
}
QLineEdit:Focus {
border: 1px solid red;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="tx_Codigo">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>205</y>
      <width>215</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background: #CFCFCF;
border-radius: 2px;
color: #000;
font: 13px &quot;Arial&quot; 
}
QLineEdit:Focus {
border: 1px solid red;
}</string>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="tx_ObsProduto">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>260</y>
      <width>450</width>
      <height>31</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background: #CFCFCF;
border-radius: 2px;
color: #000;
font: 13px &quot;Arial&quot; ;
}
QLineEdit:Focus {
border: 1px solid red;
}</string>
    </property>
    <property name="placeholderText">
     <string>Observação</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lb_FormProdutos_7">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>235</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 12px;
font-family: &quot;Arial Unicode MS&quot;;
font-weight: bold;
color: #797979
}</string>
    </property>
    <property name="text">
     <string>OBSERVAÇÃO</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_FormProdutos_8">
    <property name="geometry">
     <rect>
      <x>419</x>
      <y>290</y>
      <width>561</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 14px;
font-family: &quot;Arial&quot;;
font-weight: normal;

border-bottom: 2px solid #A2A2A2;
color: #797979
}</string>
    </property>
    <property name="text">
     <string>PREÇOS</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tx_Preco">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>360</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background: #CFCFCF;
border-radius: 2px;
color: #000;
font: 14px &quot;Arial&quot; ;
text-transform: uppercase;
}
QLineEdit:Focus {
border: 1px solid red;
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>R$ 0.00</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lb_FormProdutos_10">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>335</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 12px;
font-family: &quot;Arial Unicode MS&quot;;
font-weight: bold;
color: #797979
}</string>
    </property>
    <property name="text">
     <string>PREÇO</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tx_ValorUnitarioProduto">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>360</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background: #CFCFCF;
border-radius: 2px;
color: #000;
font: 14px &quot;Arial&quot; ;
text-transform: uppercase;
}
QLineEdit:Focus {
border: 1px solid red;
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>R$ 0.00</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lb_FormProdutos_12">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>335</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 12px;
font-family: &quot;Arial Unicode MS&quot;;
font-weight: bold;
color: #797979
}</string>
    </property>
    <property name="text">
     <string>VENDA VAREJO</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_FormProdutos_13">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>395</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 12px;
font-family: &quot;Arial Unicode MS&quot;;
font-weight: bold;
color: #797979
}</string>
    </property>
    <property name="text">
     <string>VENDA ATACADO</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tx_ValorAtacadoProduto">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>420</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background: #CFCFCF;
border-radius: 2px;
color: #000;
font: 14px &quot;Arial&quot; ;
text-transform: uppercase;
}
QLineEdit:Focus {
border: 1px solid red;
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>R$ 0.00</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="fr_BotoesFormProdutos">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>470</y>
      <width>1000</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:#E1DFE0;
border: none;</string>
    </property>
    <widget class="QPushButton" name="bt_CancelarProdutos">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>0</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
background-color: #1E87F0;
color: #FFF
 }
QPushButton:hover{
background-color: #40a286
}</string>
     </property>
     <property name="text">
      <string>CANCELAR</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/sair/delete_delete_exit_1577.png</normaloff>:/sair/delete_delete_exit_1577.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>75</width>
       <height>25</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_SalvarProdutos">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>0</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
background-color: #7AB32E;
color: #FFF
 }
QPushButton:hover{
background-color: #40a286
}</string>
     </property>
     <property name="text">
      <string>SALVAR</string>
     </property>
     <property name="icon">
      <iconset resource="../Icones/icones.qrc">
       <normaloff>:/salvar/salvar.png</normaloff>:/salvar/salvar.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lb_qtdeMin">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>350</y>
      <width>40</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="tx_AddMarca">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>145</y>
      <width>190</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background: #CFCFCF;
border-radius: 2px;
color: #000;
font: 13px &quot;Arial&quot; ;

}
QLineEdit:Focus {
border: 1px solid red;
}</string>
    </property>
    <property name="placeholderText">
     <string>NOVA MARCA</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="tx_AddCategoria">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>145</y>
      <width>190</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background: #CFCFCF;
border-radius: 2px;
color: #000;
font: 13px &quot;Arial&quot; ;

}
QLineEdit:Focus {
border: 1px solid red;
}</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="placeholderText">
     <string>NOVA CATEGORIA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_CancelAddCatergoria">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>145</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Cancelar&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background: #7AB32E;
color: #FFF
}
QPushButton:hover{
background-color: #40a286
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_CalcelAddMarca">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>145</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Cancelar&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background: #7AB32E;
color: #FFF
}
QPushButton:hover{
background-color: #40a286
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_Update_Img">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>410</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Deletar Imagem&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background: #7AB32E;
color: #FFF
}
QPushButton:hover{
background-color: #40a286
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>tx_AddMarca</zorder>
   <zorder>tx_AddCategoria</zorder>
   <zorder>bt_CancelAddCatergoria</zorder>
   <zorder>bt_CalcelAddMarca</zorder>
   <zorder>lb_FormProdutos</zorder>
   <zorder>tx_idProduto</zorder>
   <zorder>lb_FotoProduto</zorder>
   <zorder>lb_FormProdutos_2</zorder>
   <zorder>tx_DescricaoProduto</zorder>
   <zorder>lb_FormProdutos_3</zorder>
   <zorder>bt_AddCategoriaProduto</zorder>
   <zorder>lb_FormProdutos_4</zorder>
   <zorder>bt_AddMarcaProduto</zorder>
   <zorder>lb_FormProdutos_5</zorder>
   <zorder>lb_FormProdutos_6</zorder>
   <zorder>tx_EstoqueMinimoProduto</zorder>
   <zorder>tx_Codigo</zorder>
   <zorder>tx_ObsProduto</zorder>
   <zorder>lb_FormProdutos_7</zorder>
   <zorder>lb_FormProdutos_8</zorder>
   <zorder>tx_Preco</zorder>
   <zorder>lb_FormProdutos_10</zorder>
   <zorder>tx_ValorUnitarioProduto</zorder>
   <zorder>lb_FormProdutos_12</zorder>
   <zorder>lb_FormProdutos_13</zorder>
   <zorder>tx_ValorAtacadoProduto</zorder>
   <zorder>fr_BotoesFormProdutos</zorder>
   <zorder>lb_qtdeMin</zorder>
   <zorder>bt_Update_Img</zorder>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tx_idProduto</tabstop>
  <tabstop>tx_DescricaoProduto</tabstop>
  <tabstop>bt_AddCategoriaProduto</tabstop>
  <tabstop>bt_AddMarcaProduto</tabstop>
  <tabstop>tx_EstoqueMinimoProduto</tabstop>
  <tabstop>tx_Codigo</tabstop>
  <tabstop>tx_ObsProduto</tabstop>
  <tabstop>tx_Preco</tabstop>
  <tabstop>tx_ValorUnitarioProduto</tabstop>
  <tabstop>tx_ValorAtacadoProduto</tabstop>
  <tabstop>tx_AddMarca</tabstop>
  <tabstop>tx_AddCategoria</tabstop>
  <tabstop>bt_CancelAddCatergoria</tabstop>
  <tabstop>bt_CalcelAddMarca</tabstop>
 </tabstops>
 <resources>
  <include location="../Icones/icones.qrc"/>
 </resources>
 <connections/>
</ui>
